packages/core/src/mutation/create_json_mutation.ts

```diff
@@ -214,9 +214,43 @@ export function createJsonMutation(config: any): Mutation<any, any, any> {
     response: { status: config.response.status },
   });
 
+  let responseHeaders: Record<string, string> = {};
```

Comment:
🚨 it's broken:

1. In the case of two concurrent runs of _Mutation_, headers can be mixed up
2. It does not support Scope API and will be broken during SSR/tests/Storybook